autoload/neomake/makers/clippy.vim

```diff
@@ -0,0 +1,45 @@
+" vim: ts=4 sw=4 et
+
+" Yet to be determined
+let s:rustup_has_nightly = -1
+
+function! neomake#makers#clippy#clippy() abort
+    let errorfmt = '%Eerror[E%n]: %m,'.
+                 \ '%Eerror: %m,'.
+                 \ '%Wwarning: %m,'.
+                 \ '%Inote: %m,'.
+                 \ '%-Z\ %#-->\ %f:%l:%c,'.
+                 \ '%I\ %#\= %t%*[^:]: %m,'.
+                 \ '%I\ %#|\ %#%\\^%\\+ %m,'.
+                 \ '%-G%s,'
+
+    " When rustup and a nightly toolchain is installed, that is used.
+    " Otherwise, the default cargo exectuable is used. If this is not part
+    " of a nightly rust, this will fail.
+    if !exists('s:rustup_has_nightly') || s:rustup_has_nightly == -1
+        if !executable('rustup')
+            let s:rustup_has_nightly = 0
+            call system('rustc --version | grep -q "\-nightly"')
+            if v:shell_error
+                call neomake#utils#LoudMessage('Clippy requires a nightly rust installation.')
```

Comment:
Sorry, I was confusing by the levels/names.
That would show up only with level >= 2.
I think it should be at least 1 (`neomake#utils#QuietMessage`) or even `neomake#utils#ErrorMessage`.
Since it only gets displayed once, a error sounds good to me.
What do you think?